changelog.txt

```diff
@@ -2,6 +2,7 @@
 
 = 4.1.16 - 2019-xx-xx =
 * Tweak - WC 3.6 compatibility.
+* Update - Add Puerto Rico to supported countries for Payment Requests.
```

Comment:
This will need to go into 4.1.17. 4.1.16 shipped last week.

Reply:
And perhaps reword as "Enable Payment Request buttons for Puerto Rico based stores."

Reply:
Changelog updated.